README.md

```diff
@@ -1,2 +1,69 @@
-# opseng-challenge
-Wave OpsEng Development Challenge
+# Wave OpsEng Development Challenge
+
+Applicants for the [Operations Engineer career](https://www.waveapps.com/about-us/jobs/operations-engineer/) at Wave must complete the following challenge, and submit a solution prior to the interviewing process. This will help the interviewers assess your strengths, and frame the conversation through the interview process. Take as much time as you need, however we ask that you not spend more than a few hours.
+
+Send your submission to [dev.careers@waveapps.com](dev.careers@waveapps.com). Feel free to email [dev.careers@waveapps.com](dev.careers@waveapps.com) if you have any questions.
+
+## Submission Instructions
+
+1. Fork this project on github. You will need to create an account if you don't already have one
```

Comment:
@nduthoit The challenge we currently have is composed of two separate repos (one for the flask app, one for the provisioning code) - this doesn't really make sense with this instruction. Ok to modify this to reference creating your own repos, or did we want to try to update the challenge to work out of a single forked repo?